mappings/net/minecraft/item/BrushItem.mapping

```diff
@@ -1,5 +1,12 @@
 CLASS net/minecraft/class_8162 net/minecraft/item/BrushItem
+	METHOD method_49183 (Lnet/minecraft/class_1309;)V
```

Comment:
Shouldn't we perhaps try to find a name for this rather than leaving it as method_49183?

Reply:
This is a lambda.